mk/check.mk

```
@@ -42,5 +42,9 @@ ginkgo/unfocus:
 format: fmt generate docs tidy ginkgo/unfocus
 
 .PHONY: check
-check: format helm-lint golangci-lint shellcheck ## Dev: Run code checks (go fmt, go vet, ...)
+check: format helm-lint golangci-lint shellcheck kube-lint ## Dev: Run code checks (go fmt, go vet, ...)
 	git diff --quiet || test $$(git diff --name-only | grep -v -e 'go.mod$$' -e 'go.sum$$' | wc -l) -eq 0 || ( echo "The following changes (result of code generators and code checks) have been detected:" && git --no-pager diff && false ) # fail if Git working tree is dirty
+
+.PHONY: kube-lint
+kube-lint:
+	kube-linter lint .
```

```suggestion
	$(GOPATH_BIN_DIR)/kube-linter lint
```

^ ensures it will always use the kube-linter you've installed even if you have another version installed locally.

---

Also for readability it might be better to move this target above the check target (this makes check always the last one)

---

Done.